privacyidea/lib/security/aeshsm.py

```diff
@@ -22,6 +22,7 @@
 
 import binascii
 import logging
+from builtins import bytes
```

Comment:
Hm, I think not having another compatibility module in addition to six would be nice -- what do you think?

Reply:
Unfortuantely `six` does not provide the `builtins` module from Python 3 but if we change the `int_list_to_bytestring()` stuff above we don't need it anyway.

Reply:
:+1: